exp/runtime/hooks/api/v1alpha1/topologymutation_variable_types.go

```diff
@@ -16,7 +16,11 @@ limitations under the License.
 
```

Comment:
Please see your previous PR for all the places where the new builtin variable requires changes: https://github.com/kubernetes-sigs/cluster-api/pull/10822 :)